apel/db/records/job.py

```diff
@@ -146,10 +146,14 @@ def _check_factor(self, sfu, sf):
         Check for the validity of the ScalingFactorUnit and ScalingFactor fields.
         We accept neither field included or both.  If only one of the fields is 
         included, it doesn't really make sense so we reject it.
-    
-        We expect that all null values have been converted to the string 'None'.
+
+        We expect that:
```

Comment:
I changed these to make it clearer that we're expecting the right sort of arguments to the method.